build_scripts/build_usd.py

```diff
@@ -1345,20 +1345,43 @@ def InstallDraco(context, force, buildArgs):
 ############################################################
 # MaterialX
 
-MATERIALX_URL = "https://github.com/materialx/MaterialX/archive/v1.38.0.zip"
+MATERIALX_URL = "https://github.com/materialx/MaterialX/archive/v1.38.3.zip"
 
 def InstallMaterialX(context, force, buildArgs):
     with CurrentWorkingDirectory(DownloadURL(MATERIALX_URL, context, force)):
-        # USD requires MaterialX to be built as a shared library on Linux and MacOS
-        # Currently MaterialX does not support shared builds on Windows
-        cmakeOptions = []
-        if Linux() or MacOS():
-            cmakeOptions += ['-DMATERIALX_BUILD_SHARED_LIBS=ON']
+        cmakeOptions = ['-DMATERIALX_BUILD_SHARED_LIBS=ON']
 
         cmakeOptions += buildArgs;
 
+        if "v1.38.3.zip" in MATERIALX_URL:
+            # This will be fixed in v1.38.4:
```

Comment:
Perhaps comment that this is for when Python is not required.

Reply:
The default in USD is to build MaterialX without Python. This is how I found the fix was required.
I could possibly add a comment pointing to the fix if requested: https://github.com/autodesk-forks/MaterialX/pull/1344
And the fix for CMake relocability: https://github.com/autodesk-forks/MaterialX/pull/1350
That were both merged in ILM repo via: https://github.com/materialx/MaterialX/pull/805